src/frontend/efiling-frontend/src/modules/errorRedirect.js

```diff
@@ -0,0 +1,16 @@
+export function errorRedirect(errorUrl, error) {
+  if (
+    errorUrl &&
+    error.response &&
+    error.response.status &&
+    error.response.data &&
+    error.response.data.message
+  ) {
+    return window.open(
+      `${errorUrl}?status=${error.response.status}&message=${error.response.data.message}`,
```

Comment:
Could we document that in docsify, I think we need to start adding this to the getting started ?

Reply:
Updated the errorHandling.md, not sure what you envision on the docsify getting started for this?